.ci/rollback_enable_sandbox_cgroup_only.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This script is very similar to the enable one. To avoid the code duplication, I'd be tempted to merge them together into `.ci/toggle_sandbox_cgroup_only.sh` (or similar) and require the caller to specify an `enable` or `disable` argument. That will guarantee the two scripts won't diverge in future.